android/app/src/main/java/labs/pooh/eaterslab/repository/dao/FixedMenuOptionDao.kt

```diff
@@ -0,0 +1,14 @@
+package labs.pooh.eaterslab.repository.dao
+
+import labs.pooh.client.models.FixedMenuOption
+
+data class FixedMenuOptionDao(
+    val name: String,
+    val price: Double,
+    val photoUrl: String,
+    val avgReviewStars: Double,
+    val id: Int?
+) : ImageContentDao(photoUrl)
+
+fun FixedMenuOption.toDao() =
+    FixedMenuOptionDao(name, price.toDouble(), photoUrl, avgReviewStars?.toDouble() ?: 0.0, id)
```

Comment:
What happens in the interface when `avgReviewStars` is zero or empty? Is this handled?

Reply:
i tis displayed as zero stars review. can think of separate state as 'no data' to handle this but not in this PR